### ISSUE TITLE
New videos

### ISSUE THREAD

**Issue by lamador1980** [mage2tv/topics#175]
Are there any new videos coming soon? I signed up for a trial account. Your material is very good but before I commit to a subscription I want to make sure that content is being updated and that new videos are added.

BTW, are these showing empty because I'm on a trail plan, or are they empty?

![Screen Shot 2019-05-29 at 5 36 28 PM](https://user-images.githubusercontent.com/46786481/58593265-6a9d8b80-8238-11e9-80b7-5237ac47de17.png)

**Comment by Vinai** [comment]
HI @lamador1980!
Yes, new videos are coming very soon. I'm planning to move to releasing a few videos every week instead of a whole bunch at once and then nothing for a longer time.
That will be a more sustainable pace.

Regarding those categories you reference, those videos are not recorded yet.
Once more top level categories are filled I'll hide the empty categories. On the other hand, they are a nice way of showing what's planned...

Thanks for your interest!
Vinai

**Comment by lamador1980** [comment]
@Vinai 

That's great news. I feel there's a demand for this that hasn't been fully fulfilled. I'm a front-end trying to move into the backend. The Magento documentation just isn't cutting it and Magento U classes are too much money. 

From what I watched, you seem to be great at explaining each lesson, but I think there's a gap. Every lesson from start to Modularity was great. After Modularity, I could not follow because I was struggling to understand the why's of doing things.

**Comment by ryantfowler** [comment]
@Vinai Piggy-backing off of @lamador1980 's question here ... do you plan on covering those topics which he's pointed out? They've been posted on the website since it's inception, and the direction for content seems to not fall in line with those topics ... for instance there was the creation of the fundamentals section, then declarative schema, and now the graphql area.

These are all great videos, it's just that when I first saw this service I saw a list of topics which I thought would be covered by you, so that really sold me on the subscription. You are a very skilled communicator/educator of Magento knowledge, which I'm very fond of, so if it has your name on it catches my attention very quickly. Beyond the Magento-specific knowledge, you as a developer/engineer have a lot to share, so the existence of this service is great ( thank you ). As a paying subscriber, I'd also like to say "can we please have some coverage on the topics that were on the site when it was first launched?" :)

I know that this repository is meant to provide feedback to you, and that there isn't a guarantee that the videos released will correlate to the most thumbed up videos, but I'm also curious if you could provide some kind of high-level direction that you're planning on taking your videos? This way it'll inform subscribers as to the evolution of the platform.

**Comment by Vinai** [comment]
Hi @ryantfowler,

thank you for picking up this thread!
Quick answer: yes, I am planing to cover the topics he pointed out.
Regarding content creating, I'm following a two-pronged approach.
I'm trying to build up a curriculum that will help new developers become productive quickly on the Magento 2 platform, and at the same time, I want to cover valuable content for more experienced developers, too.
So after covering a lot of fundamentals (like the Dependency Injection, Declarative Schema and the JavaScript topics) I now decided to cover GraphQl because I believe that is where the most value will be. It's currently the best API to build rich store fronts. This also is part of the promised APIs section.

By the way, GraphQl is the first batch of content that currently isn't certification relevant. My guess is that will change in the future though. That is another factor I consider when choosing what to cover.

After finishing the topic on GraphQl, which is very useful but technically not very challenging, I will cover extension attributes. These are very valuable, but also very complex (and certification relevant). They are also on the list of promised topics.

Once they are covered I want to return to the fundamentals, and cover event observers, routing and action controllers next.

From that point onward I haven't decided yet what to cover next.

So to summarize, I plan to cover everything.
To do that I switch between covering advanced topics and fundamentals. When choosing which ones to cover first I try to pick the most practically valuable and also certification relevant ones.

**Comment by ryantfowler** [comment]
@Vinai thank you for providing feedback here :). I think your strategy is awesome, and I appreciate it.

Thank you!
